server/src/main/java/org/opensearch/monitor/fs/FsProbe.java

```diff
@@ -74,6 +84,10 @@ public FsInfo stats(FsInfo previous) throws IOException {
         FsInfo.Path[] paths = new FsInfo.Path[dataLocations.length];
```

Comment:
#6274 will ensure that dataLocations.length == 1 if this is a search node, right?

Reply:
Yes.